File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/room/AllRoomModel.java
package com.syahputrareno975.cardbattlemodule.model.room;

import java.io.Serializable;
import java.util.ArrayList;

public class AllRoomModel implements Serializable {
    public ArrayList<RoomDataModel> Rooms = new ArrayList<RoomDataModel>();

    public AllRoomModel() {
    }

    public AllRoomModel(ArrayList<RoomDataModel> rooms) {
        Rooms = rooms;
    }
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/room/AllRoomModel.kt
package com.syahputrareno975.simplecardbattle.model.room

import java.io.Serializable

class AllRoomModel : Serializable {
    var Rooms = ArrayList<RoomDataModel>()

    constructor()

    constructor(Rooms: ArrayList<RoomDataModel>) {
        this.Rooms = Rooms
    }
}